### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ilka-schulz** [mozzwald/framework-espidf#2]
Is this project still maintained? I am asking because the last commit is 14 months old by now, there have never been any issues in this project and the currently supported ESP toolchain is outdated (see #1).

**Comment by mozzwald** [comment]
no. you should use upstream sources. are you even building fujinet firmware?
